{{cookiecutter.root_dir__update_if_you_intend_to_use_monorepo}}/{{cookiecutter.project_name}}/validation/notebooks/ModelValidation.py

```diff
@@ -193,9 +216,14 @@ def log_to_model_description(run, success):
     )
 
 
-# run evaluate
-with mlflow.start_run() as run, tempfile.TemporaryDirectory() as tmp_dir:
+# COMMAND ----------
 
+training_run = get_training_run(model_name, model_version)
+# run evaluate
+with mlflow.start_run(
+    run_name=generate_run_name(training_run),
```

Comment:
If the training run is not found, both `run_name` and `description` will be None, which means random run name and empty description.